### PR TITLE
Rename the "sherlockml-boltzmannclean" feedstock

### PR DESCRIPTION
**Pull request by shwinnn**
This is just a rename of the existing "sherlockml-boltzmannclean" feedstock already on conda-forge. As per the [Guidelines](https://conda-forge.org/docs/orga/guidelines.html) I'm going to ping @conda-forge/core to remove the old feedstock.

### CODE REVIEW

**Comment by shwinnn** [comment]
@pbugnion @zblz

File: recipes/boltzmannclean/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
```
Comment (marcelotrevisani) [inline recipes/boltzmannclean/meta.yaml]:
```suggestion
  host:
```

File: recipes/boltzmannclean/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (marcelotrevisani) [inline recipes/boltzmannclean/meta.yaml]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

File: recipes/boltzmannclean/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - boltzmannclean
+
+about:
+  home: https://faculty.ai
+  license: Apache-2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (marcelotrevisani) [inline recipes/boltzmannclean/meta.yaml]:
```suggestion
  license_file: 'LICENSE'
```

File: recipes/boltzmannclean/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+{% set name = "boltzmannclean" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "e4b4ed0a7ae9c1f3dfb9283c87fd1c846d801934f16afcf0e09ce5f24bc8d073" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (npavlovikj) [inline recipes/boltzmannclean/meta.yaml]: Please remove the `fn` line, it is not needed anymore.

Comment (shwinnn) [reply recipes/boltzmannclean/meta.yaml]: done in c84720eec24f3112dc0a900ad739b0c2c78d948b

**Comment by shwinnn** [comment]
bump...

File: recipes/boltzmannclean/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    - numpy
+    - scipy
+    - scikit-learn
+
+test:
+  imports:
+    - boltzmannclean
+
+about:
+  home: https://faculty.ai
```
Comment (npavlovikj) [inline recipes/boltzmannclean/meta.yaml]: I would prefer if the `home` address is `https://github.com/facultyai/boltzmannclean` - the current one is too general.

**Comment by npavlovikj** [comment]
@shwinnn - I am sorry about the delay with this. I just left a minor comment - after that we can go ahead and merge the recipe.

**Comment by shwinnn** [comment]
All done, thank you for the feedback! :)

**Comment by npavlovikj** [comment]
Thanks @shwinnn !

@conda-forge/core , the package is being renamed, so can you please remove the old feedstock, https://github.com/conda-forge/sherlockml-dataclean-feedstock?

**Comment by npavlovikj** [comment]
@shwinnn , the old feedstock has been archived now.

**Comment by shwinnn** [comment]
Thanks! The package can go, yes.

**Pull request merged**